app/jobs/identities_backfill_job.rb

```diff
@@ -0,0 +1,41 @@
+class IdentitiesBackfillJob
+  # This is a short-term solution to backfill data requiring a table scan.
+  # This job can be deleted once it's done.
+
+  queue_as :low
+
+  # Let's give us the option to fine-tune this on the fly
+  BATCH_SIZE_KEY = 'IdentitiesBackfillJob.batch_size'.freeze
+  CACHE_KEY = 'IdentitiesBackfillJob.position'.freeze
+
+  def perform
+    position = Rails.cache.read(CACHE_KEY).to_i || 0
+    batch_size = Rails.cache.read(BATCH_SIZE_KEY).to_i || 500_000
+
+    # max id today is in the 184M range
+    return true if position >= 185_000_000
```

Comment:
checking the last ID is cheap, so I'd do:

```suggestion
    return true if position >= Identity.last.id
```
that way in lower environments with smaller tables, the job will still exit correctly